File: app/src/main/java/com/android/virgilsecurity/virgilback4app/util/customElements/OnFinishTimer.java
package com.android.virgilsecurity.virgilback4app.util.customElements;

import android.os.CountDownTimer;

/**
 * Created by Danylo Oliinyk on 11/27/17 at Virgil Security.
 * -__o
 */

public abstract class OnFinishTimer extends CountDownTimer {

    public OnFinishTimer(long millisInFuture, long countDownInterval) {
        super(millisInFuture, countDownInterval);
    }

    @Override public void onTick(long l) {

    }
}

File: app/src/main/java/com/android/virgilsecurity/virgilback4app/util/customElements/OnFinishTimer.kt
package com.android.virgilsecurity.virgilback4app.util.customElements

import android.os.CountDownTimer

/**
 * Created by Danylo Oliinyk on 11/27/17 at Virgil Security.
 * -__o
 */

abstract class OnFinishTimer(millisInFuture: Long, countDownInterval: Long) : CountDownTimer(
    millisInFuture,
    countDownInterval) {

    override fun onTick(l: Long) {

    }
}
